systemvm/debian/opt/cloud/bin/setup/common.sh

```diff
@@ -767,6 +769,17 @@ parse_cmd_line() {
     mv ${CHEF_TMP_FILE} /var/cache/cloud/cmd_line.json
   fi
 
+  TMP_KEY_PATH=/tmp/.auth_key
+  AUTHORIZED_KEYS_PATH=/root/.ssh/authorized_keys
+  if [ ! -z "$AUTHORIZED_KEYS" ]
+  then
+    echo "$AUTHORIZED_KEYS" > $TMP_KEY_PATH
+    base64Val=`base64 -d $TMP_KEY_PATH`
```

Comment:
minor nit - Prefer `$(command)` syntax, the "``" works but could cause issue around debugging/maintenance